src/vs/editor/contrib/suggest/suggestWidget.ts

```diff
@@ -264,6 +264,8 @@ class SuggestionDetails {
 			this.docs.appendChild(renderedContents.element);
 		}
 
+		toggleClass(this.el, 'empty-docs', this.docs.innerText.length === 0);
```

Comment:
Is the assumption that `this.docs.innerText` is empty when markdown is being used?

Reply:
Yes. `this.docs` refers to the `p` element which eventually has the `markdown-docs` class. It's `innerText` property will be empty if there is no content to show (Just like in the case shown the image below).

<img width="1227" alt="screen shot 2018-10-08 at 5 26 50 pm" src="https://user-images.githubusercontent.com/13837769/46632041-b079d800-cb1f-11e8-8ecf-537d0df04aa5.png">

Reply:
I am hesitant to add this class. On a cursory read of the code, it would lead one to believe that there are no "docs" associated with the suggestion item. In the suggestWidget file `docs` is referring to the part that appears on the right of the suggestion list.

I actually liked the change you did in your previous commit of increasing the bottom padding of `.type` all the time. Wouldnt that work?

Reply:
Yeahp. You're right. 

The problem with increasing the padding of `.type` all the time arises when the `.docs` item doesn't have content:

<img width="853" alt="screen shot 2018-10-09 at 12 49 42 am" src="https://user-images.githubusercontent.com/13837769/46646012-243ad580-cb5e-11e8-9a45-015beccdc416.png">

Here the space below the header should be smaller. See current (v1.28.0-431ef9d) aspect:

<img width="855" alt="screen shot 2018-10-09 at 12 58 24 am" src="https://user-images.githubusercontent.com/13837769/46646067-724fd900-cb5e-11e8-8567-cbfe6ab268e3.png">

I think I just found an approach that doesn't need the `empty-docs` class.

Reply:
@ramya-rao-a I think the last commit solves this last issue without adding the `empty-docs` class. All the cases detailed in this comment https://github.com/Microsoft/vscode/pull/59786#issuecomment-427958790 are still working.

PS. Do you mind if I squash the commits in this PR?